File: examples/workspaces/booksandrecords-spring-boot/src/main/scala/com/ms/booksandrecords/service/dealevent/DealOpened.scala
package com.ms.booksandrecords.service.dealevent

import com.ms.booksandrecords.service.DealEvent

class DealOpened(var id: String, var product: String, var price: Float, var quantity: Int) extends DealEvent {
  def getId = id

  def setId(id: String) = this.id = id

  def getProduct = product

  def setProduct(product: String) = this.product = product

  def getPrice = price

  def setPrice(price: Float) = this.price = price

  def getQuantity = quantity

  def setQuantity(quantity: Int) = this.quantity = quantity
}

File: examples/AppModel/booksandrecords-spring-boot/src/main/java/com/ms/booksandrecords/service/dealevent/DealOpened.java
package com.ms.booksandrecords.service.dealevent;

import com.ms.booksandrecords.service.DealEvent;

public class DealOpened extends DealEvent {
    private String id;
    private String product;
    private float price;
    private int quantity;
    public DealOpened(String id, String product, float price, int quantity){
        this.id = id;
        this.product = product;
        this.price = price;
        this.quantity = quantity;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getProduct() {
        return product;
    }

    public void setProduct(String product) {
        this.product = product;
    }

    public float getPrice() {
        return price;
    }

    public void setPrice(float price) {
        this.price = price;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}
